docs/layout.html

```diff
@@ -38,5 +39,62 @@ <h1><img src="logo.svg"> Mithril <small>[version]</small></h1>
 			}
 		}
 		</script>
+		<script src="https://flems.io/flems.html" defer></script>
+		<style type="text/css" media="screen">
+			.flems main {
+				margin: 0;
+				max-width: auto;
+			}
+			.flems {
+				margin: 20px;
+				max-height: 400px;
+				box-shadow: 0 0 20px rgba(0,0,0,0.25);
+			}
+			@media (max-width: 500px) {
+				.flems {
+					min-height: calc(100vw * 1.3);
+				}
+			}
+		</style>
+		<script>
+			"use strict"
+			var script = document.createElement("script")
+			script.onload = function() {
+				var systemFonts = [
+					"body { font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'; }",
+					"body { height: 100%; overflow:hidden; display:flex; justify-content: center; align-items:center; }"
+				].join("")
+				;[].forEach.call(document.querySelectorAll("pre.language-js,pre code.lang-js"), function(el) {
+					el.tagName === "CODE" && (el = el.parentNode)
```

Comment:
So it's a little more readable:

```suggestion
 					if (el.tagName === "CODE") el = el.parentNode
```